admin_manual/configuration_server/occ_command.rst

```diff
@@ -186,6 +187,25 @@ you need to specify ``--program occ`` after the ``--generate-hook``.
 If you want the completion to apply automatically for all new shell sessions, add the command to your
 shell's profile (eg. ``~/.bash_profile`` or ``~/.zshrc``).
 
+.. _run_commands_in_maintenance_mode:
+
+Run commands in maintenance mode
+--------------------------------
+
+In maintenance mode, no apps are loaded¹.
+
+Commands provided by apps are not available.
+
+The commands, available in maintenance mode, are integrated in Nextcloud server.
+A command may use events to communicate with other apps. An app can only react to an event when loaded. 
+
+Example: The command `user:delete` deletes a user account. UserDeletedEvent is emitted. Calendar app implements an event listener to delete user data². In maintenance mode, the Calendar app is not loaded, and hence the user data not deleted.
+
+Consider running such commands with maintenance mode off.
+
+| ¹ Exception: `The settings app is loaded <https://github.com/nextcloud/server/blob/75f17b60945e15effc3eea41393eef2b13937226/lib/base.php#L780>`_
+| ² `Calendar app event listener for UserDeletedEvent <https://github.com/nextcloud/calendar/blob/87e8586971a8676dc15a90f0cd969274678b7009/lib/Listener/UserDeletedListener.php>`_
```

Comment:
https://docutils.sourceforge.io/docs/user/rst/quickref.html#footnotes

Reply:
Thanks :+1: 

![image](https://user-images.githubusercontent.com/3902676/234370818-c650d934-ab3d-44a0-9fc7-91f0d77cdbfa.png)

I like the styling a bit less. It's cool that the ¹ is an actual link. 

I don't have a strong opinion. Which one do you prefer? 

If you decide for real footnotes, do you want me to update system tags https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/occ_command.html#system-tags too?

Reply:
I prefer clickable for the pdf version

Reply:
Real footnotes it is :partying_face: